app/src/test/java/com/example/memeapp/presentation/mvp/presenter/MemeAppPresenterTest.kt

```diff
@@ -57,11 +57,11 @@ class MemeAppPresenterTest {
 
     @Test
     fun `load local database data`() {
-        whenever(memeAppModel.loadMemesFromDataBase()).thenReturn((memes))
-        memeAppPresenter.getMemesFromDataBase()
+        whenever(memeAppModel.loadMemesFromDatabase()).thenReturn((memes))
```

Comment:
why this double parentesis `thenReturn((memes))` is this working?

Reply:
Now its no more, thx